### PR TITLE
Add french translations and (very) minor rewording to English

### PR DESCRIPTION
**Pull request by RannyBergamotte**
All the translations are (hopefully) done.

Note: did some testing, and seems like french translations don't load? (screenshot below is also set to french)

Also not sure anymore about the english rewording of `Show Expression List Top Bar` in en.ftl line 79, as it makes the words go to the next line. I just found `show top bar` confusing as initially I thought it means the top black bar with the logo.
image

Also nice new ui for the categories!

### CODE REVIEW

**Comment by RannyBergamotte** [comment]
Welp, french translations do load fine now, forgot to update the i18n-core file, will investigate prettier failing and ui fix for overflowing text in desmodder options and some weirdness in video creator menu. Might possibly need some help on ui fixing.

File: localization/en.ftl [hunk 1]
```diff
@@ -76,7 +76,7 @@ builtin-settings-opt-expressions-name = Show Expressions
 builtin-settings-opt-expressions-desc = {""}
 builtin-settings-opt-zoomButtons-name = Show Zoom Buttons
 builtin-settings-opt-zoomButtons-desc = {""}
-builtin-settings-opt-expressionsTopbar-name = Show Top Bar
+builtin-settings-opt-expressionsTopbar-name = Show Expression List Top Bar
```
Comment (jared-hughes) [inline localization/en.ftl:79]: Quick change to get it to fit. Does this wording make sense?
```suggestion
builtin-settings-opt-expressionsTopbar-name = Show Expressions Top Bar
```

File: localization/fr.ftl [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+wolfram2desmos-opt-reciprocalExponents2Surds-name = Notation Radical
+wolfram2desmos-opt-reciprocalExponents2Surds-desc = Conversion de puissances fractionnaires inférieures powers à un en radical équivalent (surd)
+wolfram2desmos-opt-derivativeLoopLimit-name = Développer les Dérivés
+wolfram2desmos-opt-derivativeLoopLimit-desc = Développer les nth dérivés de notation Leivniz en dérivés répétés (limité à 10)
+
+## Pin Expressions
+pin-expressions-name = Épingler l'Expression
+pin-expressions-desc = Épingler l'Expression depuis Edit List mode
+pin-expressions-pin = Épingler
+pin-expressions-unpin = Épingler
```
Comment (jared-hughes) [inline localization/fr.ftl:60]: Should this be the same as -pin? Google suggests détacher.

File: localization/fr.ftl [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+## Wakatime
+wakatime-name = WakaTime
+wakatime-desc = Suivez votre activité sur Desmos sur WakaTime.com
+wakatime-opt-secretKey-name = Clef Secrète
+wakatime-opt-secretKey-desc = Clé API utilisée pour les serveurs WakaTime
+wakatime-opt-splitProjects-name = Diviser les projets par graphique
+wakatime-opt-splitProjects-desc = Stockez chaque graphique comme son propre projet au lieu des branches d'un projet unifié
+wakatime-opt-projectName-name = Nom du projet
+wakatime-opt-projectName-desc = Visible from WakaTime, and shared for all Desmos projects
```
Comment (jared-hughes) [inline localization/fr.ftl:157]: Untranslated

File: localization/fr.ftl [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+## Performance Display
+performance-info-name = Performance Display
```
Comment (jared-hughes) [inline localization/fr.ftl:160]: Untranslated

File: localization/fr.ftl [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+performance-info-desc = Affiche des informations sur les performances du graphique actuel.
+performance-info-refresh-graph = Actualiser le graphique
+performance-info-refresh-graph-tooltip = Actualisez le graphique pour mesurer le temps de chargement initial
+performance-info-sticky-tooltip = Garder le Menu Ouvert
+performance-info-time-in-worker = Time In Worker
```
Comment (jared-hughes) [inline localization/fr.ftl:165]: Untranslated

File: localization/fr.ftl [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+performance-info-compiling = Compilation
+performance-info-rendering = Rendering Rendition
```
Comment (jared-hughes) [inline localization/fr.ftl:167]: Is this really two words? Google translate directly translates "rendering" as "le rendu"

File: localization/fr.ftl [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+performance-info-other = Autre
+
+## Better Evaluation View
+better-evaluation-view-name = Meilleure Vue d'évaluation
+better-evaluation-view-desc = Displays list elements, colors, and undefined values
```
Comment (jared-hughes) [inline localization/fr.ftl:172]: Untranslated

File: localization/fr.ftl [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+video-creator-ffmpeg-fail = Si cela ne fonctionne pas dans quelque instants, essayez de recharger la page ou de signaler ce bogue aux développeurs de DesModder.
+video-creator-exporting = Rendition en cours...
+video-creator-cancel-capture = Annuler
+video-creator-cancel-export = Annuler
+video-creator-capture = Enregistrer
+video-creator-preview = Avant-première
+video-creator-delete-all = Tout Suprimer
+video-creator-filename-placeholder = définir le nom de fichier
+video-creator-export = Créer
+video-creator-export-ff = Créer (Avertissement : actuellement peu fiable/lent dans Firefox)
```
Comment (jared-hughes) [inline localization/fr.ftl:138]: `video-creator-export-ff` is no longer used

File: localization/fr.ftl [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+
+## Category names
+category-core-name = Essentiel
+category-utility-name = Utilitaire
+category-visual-name = Visuel
+category-integrations-name = Intégrations
+
+## GLesmos
+GLesmos-name = GLesmos
+GLesmos-desc = Rendre les implicites sur le GPU. Désactive sur rechargement de l'onglet. Dans des rares cas, peut causait l'onglet de s'arreter; reload the page if you have issues.
```
Comment (jared-hughes) [inline localization/fr.ftl:22]: End of this string is untranslated

File: localization/fr.ftl [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+builtin-settings-name = Paramètres
+builtin-settings-desc = Permet l'utilisation de fonctionnalités intégrées à Desmos. La plupart des option appliquent uniquement à votre navigateure et sont ignorées lorsque vous partagez des graphiques avec d'autres.
+builtin-settings-opt-advancedStyling-name = Outils de Style Avancé
+builtin-settings-opt-advancedStyling-desc = Permet l'édition d'étiquettes, l'affichage au survol, le contour du texte et la grille à un quadrant
+builtin-settings-opt-graphpaper-name = Tableau Graphiques
+builtin-settings-opt-graphpaper-desc = {""}
+builtin-settings-opt-administerSecretFolders-name = Créer des dossiers cachés
+builtin-settings-opt-administerSecretFolders-desc = {""}
+builtin-settings-opt-pointsOfInterest-name = Afficher les points d'intérêt
+builtin-settings-opt-pointsOfInterest-desc = Interception, trous , intersections, etc.
```
Comment (jared-hughes) [inline localization/fr.ftl:72]: Excessive space before the comma here?

File: localization/fr.ftl [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+pin-expressions-unpin = Épingler
+
+## Builtin Settings
+builtin-settings-name = Paramètres
+builtin-settings-desc = Permet l'utilisation de fonctionnalités intégrées à Desmos. La plupart des option appliquent uniquement à votre navigateure et sont ignorées lorsque vous partagez des graphiques avec d'autres.
+builtin-settings-opt-advancedStyling-name = Outils de Style Avancé
+builtin-settings-opt-advancedStyling-desc = Permet l'édition d'étiquettes, l'affichage au survol, le contour du texte et la grille à un quadrant
+builtin-settings-opt-graphpaper-name = Tableau Graphiques
+builtin-settings-opt-graphpaper-desc = {""}
+builtin-settings-opt-administerSecretFolders-name = Créer des dossiers cachés
```
Comment (jared-hughes) [inline localization/fr.ftl:69]: Renamed to `builtin-settings-opt-authorFeatures-name`

**Review by jared-hughes — COMMENTED** [review]
Looks good in general! Thanks for all the great translations. I put some review comments. Mostly phrases that seem untranslated. If there's no reasonable translation into French, put a comment saying that.

Looks like it's missing the following two keys:
 - `video-creator-method-ticks`
 - `video-creator-ticks-step`

File: localization/fr.ftl (hunk 5 above)
Comment (RannyBergamotte) [reply localization/fr.ftl:165]: I have no idea for the translation for Time in Worker - I don't know how worker would be translated. I'll look into this, but may just end up leaving it as is

**Pull request merged**